src/pages/ReportSettingsPage.js

```diff
@@ -217,6 +225,41 @@ class ReportSettingsPage extends Component {
                                 </OfflineWithFeedback>
                             </View>
                         )}
+                        <View>
```

Comment:
Is the nested View necessary?

Reply:
It's not needed, I'll remove it. Thanks!